app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,24 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            hummer_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self.nickname = nickname
+        self._favourite_dish = favourite_dish
```

Comment:
same, fix this for all classes
why do u need to recreate attributes from parent class in child ?